src/test/regress/expected/failure_savepoints.out

```diff
@@ -66,16 +66,26 @@ SELECT citus.mitmproxy('conn.onQuery(query="^RELEASE").kill()');
 (1 row)
 
 BEGIN;
-INSERT INTO artists VALUES (5, 'Asher Lev');
+UPDATE artists SET name='a';
 SAVEPOINT s1;
 DELETE FROM artists WHERE id=4;
 RELEASE SAVEPOINT s1;
+WARNING:  AbortSubTransaction while in COMMIT state
```

Comment:
This means `RELEASE SAVEPOINT` started, but didn't finish, so this warning is expected.